acquia/blt
# Fixes #3148: Frivolous composer notices.

Fixes #3148 
--------

Changes proposed
---------
- Add a post-autoload-dump script to be invoked by the project template, which will re-sort Composer's install.json file to ensure that composer/installers gets loaded prior to blt on subsequent composer operations.

Testing steps
----------
1. Starting from a normal BLT-based project, run any composer command (`composer --version`) and get all of the notices about "could not scan for classes...".
2. Apply this patch via Composer: https://patch-diff.githubusercontent.com/raw/acquia/blt/pull/3701.diff (note you MUST use the .diff format, not .patch like normal)
3. Run any composer command (`composer --version`) and verify that the notices are no longer generated. 

Additional details
-----------
I would consider this a workaround until/if this is fixed upstream: https://github.com/composer/composer/issues/8065
